yaml/pois.yaml

```diff
@@ -355,8 +355,13 @@ filters:
     output:
       kind: {col: tourism}
   - filter:
-      amenity: [atm, bicycle_rental, bicycle_parking, bus_stop, drinking_water, emergency_phone,
-        parking, post_box, telephone, theatre, toilets]
+      amenity: [theatre]
```

Comment:
Split theatre off to it's own filter as it should really be visible earlier than these "infrastructure" related things, and considered separately from.